File: medium_exercises/python_shell/exe4.py
#!/usr/bin/env python3
import os

file='/home/kmullen/bash_scripts'

if os.path.exists(file):
    print(f'{file} passwords are enabled')

if os.access(file, os.R_OK):
    print(f"You have premission to edit {file}")
else:
    print(f'You do not have premission to edit {file}')

File: medium_exercises/ruby_shell/exe4.rb
#! /bin/ruby

file = "/home/kmullen/bash_scripts"

if File.exist?(file)
  puts "#{file} passwords are enabled"
end

if File.writable?(file)
  puts "You have permissions to edit #{file}"
else
  puts "You do not have permissions to edit #{file}"
end


